src/components/sidebar-entities/settings-sidebar/style.scss

```diff
@@ -0,0 +1,9 @@
+@import '~styles/theme';
+
+.SettingsSidebar-Content {
+  margin: 0 1rem;
+
+  .LabeledItem-Label {
```

Comment:
Styles of one component shouldn't be modify styles of other components. To apply this you should pass a style to the component. Or add a parameter to the component that adds this style.

Reply:
Fixed in Irame/sectors-without-number@ec5d60006c0216ae4bd29cec5d4b31fb45bd5967